docs/source/configuration.rst

```diff
@@ -291,7 +291,7 @@ above:
 .. code-block:: cfg
 
     [sqlfluff]
-    warnings = L019, L006
+    warnings = L019, LT01
```

Comment:
Oh interesting, I didn't know we were downgrading this to a warning

Reply:
... we're not - it's just an example 😄

Reply:
🙃